invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/CreatibutorsField/CreatibutorsField.js

```diff
@@ -11,14 +11,20 @@ import PropTypes from "prop-types";
 import { getIn, FieldArray } from "formik";
 import { Button, Form, List, Icon } from "semantic-ui-react";
 import _get from "lodash/get";
-import { FeedbackLabel, FieldLabel } from "react-invenio-forms";
+import {
+  FeedbackLabel,
+  FieldLabel,
+  createCommonDepositFieldComponent,
+  mandatoryFieldCommonProps,
```

Comment:
may i ask where `mandatoryFieldCommonProps` is defined in `react-invenio-forms`. i can't find it.

Reply:
ahhh, thanks, i thought i checked all PR's from you in react-invenio-forms. i missed that one.